events/2021-kubevirt-summit/proposals/hco-proposal-2021.md

```diff
@@ -0,0 +1,28 @@
+# KubeVirt HyperConverged Cluster Operator: What is New, What to Expect
+
+# Abstract
+The [HyperConverged Cluster operator](https://github.com/kubevirt/hyperconverged-cluster-operator) (HCO) is a unified operator
+deploying and controlling KubeVirt and several adjacent operators:
+* Containerized Data Importer
+* Scheduling, Scale and Performance
+* Cluster Network Addons
+* Node Maintenance
+
+The purpose of HCO is to ease the deployment, upgrade, monitoring and configuration of an opinionated version of the he KubeVirt cluster.
+
+In this session will cover the following topics:
+1. Why we think that an opinionated deployment and a single source of truth for the whole product is really important
```

Comment:
Would this point belong more in #57 which covers HCO for the user?

Well, certainly a short overview of the background is welcome, I'm just wondering about potential [too much] overlap here